parameter/FloatingPointParameterType.cpp

```diff
@@ -138,7 +139,8 @@ bool CFloatingPointParameterType::toBlackboard(
             return false;
         }
 
-        float fData = reinterpret_cast<const float &>(uiValue);
+        float fData;
+        utility::binaryCopy(uiValue, fData);
```

Comment:
I already considered this too. But I know from a previous try at another patch that Visual Studio 12 does not support using `decltype` on the variable being declared (which is the case in your suggestion). :(